packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,20 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## September
 
+### Weekly release 2022.09.3
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.09.3](#bugs-fixed-in-2022-09-3)                      | September 28, 2022            |
+
+#### Bugs fixed in 2022.09.3
+
+* Existing provisioning settings for an app were reset to `None` when an app was updated using PUT (`/apps/${applicationId}`). (OKTA-520647)
```

Comment:
```suggestion
* Existing provisioning setting for an app was reset to `None` when an app was updated using `PUT /apps/${applicationId}`. (OKTA-520647)
```
Should we change this to singular "setting" since there's just one setting that was reset?